cms_delete_content/static/src/js/delete_content.js

```diff
@@ -0,0 +1,58 @@
+odoo.define('cms_delete_content.delete_confirm', function (require) {
+    'use strict';
+
+    var ajax = require('web.ajax');
+    var msg_tool = require('cms_status_message.tool');
+    var core = require('web.core');
+    var _t = core._t;
+
+    $(document).ready(function () {
+      $(".cms_delete_confirm").click(function(evt){
```

Comment:
Besides following the Odoo standard, which is a benefit by itself (you see one animation and you know how they all work), [you get all of this](https://github.com/odoo/odoo/blob/9.0/addons/web_editor/static/src/js/snippets.animation.js), among which there are:

- No need for `$(document).ready()`.
- You inherit/extend throught `web.Class`.
- `this.$`
- Auto-applied to all elements matching `this.selector`.